compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -1165,6 +1165,11 @@ impl<'a> Parser<'a> {
         let open_paren = self.token.span;
 
         let mut seq = self.parse_paren_expr_seq().map(|args| {
+            // njn: this is assuming that the function name is in a single
+            // token, and that `pref_token.span` covers that. But if the
+            // function name was expanded from a macro, it could have the form
+            // `«`, ident, `»`, and so prev_token.span doesn't go back far
+            // enough
```

Comment:
Then the starting span has to be recorded at some earlier point, this is a pretty common situation.